.github/workflows/test-plan.yml

```diff
@@ -72,6 +72,23 @@ jobs:
         with:
           ref: '${{ github.event.pull_request.head.ref }}'
 
+      - name: 'Setup Go'
+        uses: 'actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568' # ratchet:actions/setup-go@v3
+        with:
+          go-version: '1.20'
+
+      - name: 'Install Guardian CLI'
+        shell: 'bash'
+        run: 'go install github.com/abcxyz/guardian/cmd/guardian@${{ github.event.pull_request.head.sha }}'
+
+      - name: 'Setup Terraform'
+        uses: 'abcxyz/secure-setup-terraform@78e3f2fdfb7ebf59ce42141f6dd93bdd65bf04ed' # ratchet:abcxyz/secure-setup-terraform@v0.2.5
+        with:
+          terraform_version: '1.3.6'
```

Comment:
Should this be configurable (can be a follow-up)?

Reply:
Do you mean as a matrix strategy for testing purposes? Otherwise, this is our test workflow we can update as needed.